src/components/RefinementList.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <div :class="suit('')" v-if="state">
+    <slot
+      :items="state.items"
+      :refine="state.refine"
+      :searchable="searchable"
```

Comment:
Don't think we need to provide the value back since the user already provide it.